### PR TITLE
add condition to pass the check of postal code for countries that don…

### PR DESCRIPTION
**Pull request by yij793**
address: https://nanopay.atlassian.net/browse/NP-3151
Screen Shot 2021-01-06 at 1 02 54 PM

### CODE REVIEW

File: src/foam/nanos/auth/Address.js [hunk 1]
```diff
@@ -223,10 +223,18 @@ foam.CLASS({
...
+              e.EQ(foam.nanos.auth.Address.COUNTRY_ID, 'TT'),
+              e.EQ(foam.nanos.auth.Address.COUNTRY_ID, 'MY'),
+              e.EQ(foam.nanos.auth.Address.COUNTRY_ID, 'MX'),
+              e.EQ(foam.nanos.auth.Address.COUNTRY_ID, 'JM'),
+              e.EQ(foam.nanos.auth.Address.COUNTRY_ID, 'LB'),
+              e.GT(
+                foam.mlang.StringLength.create({
+                  arg1: foam.nanos.auth.Address.POSTAL_CODE
+                }), 0)
+            )
```
Comment (minsunny) [inline src/foam/nanos/auth/Address.js:237]: question! We already check the postal code validations on the each configured country. Then We need this here?

**Comment by zahnno** [comment]
If we are missing regions from countries we support, we should add them. Along with zip code validations.

File: src/foam/nanos/auth/Address.js (hunk 1 above)
Comment (yij793) [reply src/foam/nanos/auth/Address.js:237]: yeah, they want 2 error message, one is if it is empty, one is invalid

**Comment by jlhughes** [comment]
@zahnno I agree.  @yij793 can you create a ticket to capturing all the missing regions and code validations.  Then work on it slowly (lower priority, as time permits), country by country.

**Pull request closed without merging**